src/Tribe/Views/V2/Views/By_Day_View.php

```diff
@@ -715,17 +693,12 @@ protected function build_url_for_date( $date, $canonical, array $passthru_vars =
 	 * @return array<int,\stdClass> A map from each event Post ID to the value object that will represent
 	 *                              the Event ID, start date, end date and timezone.
 	 */
-	protected function prepare_day_results( array $view_event_ids, $use_site_timezone ) {
+	protected function prepare_day_results( array $view_event_ids ) {
```

Comment:
deprecate the removed param.
Also: add a `@since` to the docblock for the change.